server/clients/manageRecallsApi/updateRecall.pact.test.ts

```diff
@@ -44,7 +44,7 @@ pactWith({ consumer: 'manage-recalls-ui', provider: 'manage-recalls-api' }, prov
         probationOfficerName: 'Dave Angel',
         probationOfficerPhoneNumber: '07473739388',
         probationOfficerEmail: 'probation.office@justice.com',
-        probationDivision: 'LONDON',
+        localDeliveryUnit: 'LONDON',
```

Comment:
@jonnywyatt This usage also has to be a valid value of course.

Reply:
Good spot, corrected